src/__tests__/Flash.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import {Flash} from '..'
 import {COMMON} from '../constants'
 import theme, {colors} from '../theme'
+import {default as primitives} from '@primer/primitives'
```

Comment:
not sure exactly how theme-preval works, but it's not getting the merged colors/shadows on import, so I'm importing primitives instead. Maybe feels like there might be a better solution here?

Reply:
I think the easier way to write these tests would be to use the `theme` object instead of importing from primitives.

```diff
- primitives.colors.light.alert.warn.bg
+ theme.alert.warn.bg
```

Reply:
```suggestion
```